internal/secrets/localsecrets/localsecrets.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package localsecrets provides a way to encrypt and decrypt small messages
+// without making network calls to a third party service.
+package localsecrets
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"io"
+
+	"github.com/google/go-cloud/internal/secrets/driver"
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// secretKeeper holds a secret for use in symmetric encryption,
```

Comment:
I'd like to clarify this in general: this should be a `keyKeeper` (or `keyper`? lol) Basically a "secret" is the actually message that is going to be protected by a "key". We should also make it clear that the secrets package does not hold the secret. Maybe "secrets" is not a good name sigh.

Reply:
In reading background information about cryptography, it seems like "secret", "shared secret" and "secret key" are all used. In the case of the local implementation, it actually does hold the "secret"/secret key. Not all implementations of the unified `Crypter` interface would have to be `secretKeeper`s, but this one is.

That being said, if folks think `KeyKeeper` is less confusing, I don't feel strongly about it.

Reply:
My question is not about what the meaning behind each name is. We have two kinds of things, one used to encrypt a plain text to a cipher text, one is the cipher text itself. My comment is to figure out a general naming for these two things.

Reply:
I'm not sure I'm totally following if you have a comment/requested edit, or a question.